packages/react/src/Icons/Icon.client.stories.tsx

```diff
@@ -0,0 +1,35 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Headphones } from "@mui/icons-material";
+import * as Icons from "@mui/icons-material";
+import "./Icon.client.stories.css";
+
+const meta: Meta = {
+  title: "Look'n feel/Icons",
+};
+
+export default meta;
+
+export const Example: StoryObj = {
+  name: "Icon",
+  render: ({ ...args }) => <Headphones {...args} />,
+  args: {},
+};
+
+export const AllIcons: StoryObj = {
+  render: () => {
+    return (
+      <ul className="icon-list">
+        {Object.keys(Icons).map((x) => {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          const Comp = (Icons as any)[x];
```

Comment:
normalement tu peux mettre sa pour éviter le any
```suggestion
          const Comp = (Icons as ReactElement)[x];
```

Reply:
non il veut pas ;( 
![image](https://github.com/AxaFrance/design-system/assets/35838907/a4d07515-89da-4b9f-b98a-2aef80b19b89)

Reply:
oups c'est un tableau

```suggestion
          const Comp = (Icons as ReactElement[])[x];
```

Reply:
c pareil on s'en sort pas :( et si tu rajoute un unknown devant comme recommende, tarrive a :

![image](https://github.com/AxaFrance/design-system/assets/35838907/2d8515c8-25b9-4f9e-ac5a-94cd04ba53fb)

Reply:
j'ai fait autrement, c plus joli :)